src/pages/about.vue

```diff
@@ -1,13 +1,57 @@
 <template>
-    <div class="page-text">
-        <p v-if="appConfig" class="ma-4 text-body-1">
-            {{ appConfig.title }} is in development. Check back later for more about the application.
-        </p>
-    </div>
+    <!-- Light theme for this page so that the logo images have enough contrast to be legible -->
+    <v-theme-provider theme="light" with-background>
+        <v-container class="page-text" style="min-height: 100vh">
+            <v-col>
+                <v-row>
+                    <h1 class="text-h4">About this site</h1>
+                    <p class="text-body-1">
+                        This is an interactive map tool developed to allow users to access and explore high resolution
+                        dengue transmission intensity in countries where dengue is endemic. These maps were generated by
+                        fitting environmentally driven geospatial models to force of infection estimates derived from
+                        surveillance data including case-notification data and cross-sectional serological surveys. We
+                        provide estimates of:
+                    </p>
+                    <ul class="text-body-1">
+                        <li>
+                            The average dengue force of infection (i.e. the per capita annual risk of dengue infection
+                            for a susceptible individual)
+                        </li>
+                        <li>The seroprevalence at 9 years of age</li>
+                        <li>The annual expected number of hospital admissions by age</li>
+                        <li>
+                            Estimates of these metrics are available at the first and second administrative unit level.
+                        </li>
+                    </ul>
+                    <p class="text-body-1">We gratefully acknowledge support for this work from XXX.</p>
```

Comment:
Let's just take this para out for now but make a ticket to add it back in when we have the details,

Reply:
Created the ticket: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5277/Fill-in-acknowledgements-in-about-page